esp-hal-common/src/lib.rs

```diff
@@ -17,6 +17,7 @@
 //! [esp32s3-hal]: https://github.com/esp-rs/esp-hal/tree/main/esp32s3-hal
 
 #![no_std]
+#![feature(asm_experimental_arch)]
```

Comment:
```suggestion
#![cfg_attr(target_arch = "xtensa", feature(asm_experimental_arch)]
```

This should let us continue to build using the `stable` channel for RISC-V, I think.